src/msg.fbs

```diff
@@ -247,13 +253,6 @@ table StatRes {
   has_mode: bool; // false on windows
 }
 
-table WriteFileSync {
-  filename: string;
-  data: [ubyte];
-  perm: uint;
-  // perm specified by https://godoc.org/os#FileMode
-}
-
```

Comment:
Good catch - thanks - can you remove this in a standalone commit?

Reply:
Sure